datastore/src/main/kotlin/com/openlattice/entitysets/controllers/EntitySetsController.kt

```diff
@@ -320,7 +320,7 @@ constructor(
         )
         timer.reset().start()
 
-        deletionManager.authCheckForEntitySetAndItsNeighbors(entitySetId, deleteType, Principals.getCurrentPrincipals())
+        deletionManager.authCheckForEntitySetsAndNeighbors(setOf(entitySetId), DeleteType.Hard, Principals.getCurrentPrincipals())
```

Comment:
We specifically made the change to remove `DeleteType.Hard` as being the default. This should remain as `deleteType`; please revert.